owners/src/github.js

```diff
@@ -229,6 +229,46 @@ class GitHub {
     );
   }
 
+  /**
+   * Requests a review from GitHub users.
+   *
+   * @param {number} number PR number.
+   * @param {string[]} reviewers the list of usernames to request reviews from.
+   */
+  async createReviewRequests(number, reviewers) {
+    if (!reviewers.length) {
+      this.logger.warn(
+        `Attempted to request reviews for PR ${number} ` +
+          'but provided an empty username list'
+      );
+      return;
+    }
+    this.logger.info(
+      `Requesting review for PR #${number} from: ${reviewers.join(', ')}`
+    );
+
+    await this.client.pullRequests.createReviewRequest(
+      this.repo({number, reviewers})
+    );
+  }
+
+  /**
+   * Retrieves code review requests for a PR from GitHub.
+   *
+   * @param {number} number PR number.
+   * @return {string[]} the list of code reviews.
+   */
+  async getReviewRequests(number) {
+    this.logger.info(`Fetching review requests for PR #${number}`);
+
+    const response = await this.client.pullRequests.listReviewRequests(
+      this.repo({number})
+    );
+    this.logger.debug('[getReviewRequests]', number, response.data);
+
+    return response.data.users.map(({login}) => login);
```

Comment:
If you ever want to make the head of a 50+ y/o programmer explode, show them this line

Reply:
lol

Reply:
lol